t/ui/14-dashboard-parents.t

```diff
@@ -109,6 +109,7 @@ ok($element->is_displayed(), 'link to child group expanded');
 # first try of click does not work for unknown reasons
 for (0 .. 10) {
     $driver->find_element_by_link_text('Build0091')->click();
+    sleep 1;
```

Comment:
Likely `disable_bootstrap_animations();` does not work reliably. Have you tried fixing it? Otherwise I would at least add a note why the sleep is required.

Reply:
Or simply use `wait_until_element_gone` here for a more elegant workaround.

Reply:
I suggest approving this PR to make test pass and create ticket for mentioned optimization, (unless we have better fix already).
Both tests were repeated 10 times in CI to make sure that the fix really fixes the problem
https://circleci.com/gh/os-autoinst/openQA/4482?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link